src/react-themer/index.js

```diff
@@ -33,17 +34,23 @@ export default (customThemer: ?Object) => (theme?: Object) => (component: React.
   let resolvedAttrs;
   let themesToResolve;
 
-  return class extends Component {
+  class Main extends Component {
     static displayName = `Themer(${getDisplayName(rawThemerAttrs.component) || 'Component'})`;
     static rawThemerAttrs = rawThemerAttrs;
 
+    static PropTypes = {};
+
     static contextTypes = {
       theme: PropTypes.object,
     };
 
     componentWillMount() {
       const { theme: globalTheme } = this.context;
 
+      each(themerInstance.getVariants(), (value, key) => {
+        Main.PropTypes[key] = React.PropTypes.bool;
+      });
```

Comment:
@bevanhunt I'm just taking a bit of a closer look. I think it makes more sense here to use the `ES6` `for` loop here, so you can avoid importing anything from `lodash`. Maybe try something like: 
```      
for (const key of themerInstance.getVariants()) {
  Main.PropTypes[key] = React.PropTypes.bool;
}
```